packages/yoroi-extension/app/styles/themes/revamp/light-theme-base.js

```diff
@@ -0,0 +1,90 @@
+// @flow
+
+const cyan = { '400': '#59B1F4', '100': '#F2F9FF' };
+const yellow = { '500': '#ECBA09', '100': '#FDF7E2' };
+const magenta = {
+  main: '#FF1351',
+  '700': '#CF053A',
+  '600': '#E80742',
+  '500': '#FF1351',
+  '300': '#FBCBD7',
+  '100': '#FFF1F5',
+};
+
+export const lightTheme = {
+  name: 'revamp',
+  palette: {
+    mode: 'light',
+    /* `main` is added since MUI required it but we don't use it at all */
+    primary: {
+      main: '#4B6DDE',
+      '900': '#121F4D',
+      '800': '#122770',
+      '700': '#1737A3',
+      '600': '#3154CB',
+      '500': '#4B6DDE',
+      '400': '#7892E8',
+      '300': '#A0B3F2',
+      '200': '#C4CFF5',
+      '100': '#E4E8F7',
+      contrastText: '#FFF',
+    },
+    secondary: {
+      main: '#16E3BA',
+      '900': '#17453C',
+      '800': '#12705D',
+      '700': '#0B997D',
+      '600': '#08C29D',
+      '500': '#16E3BA',
+      '400': '#66F2D6',
+      '300': '#93F5E1',
+      '200': '#C6F7ED',
+      '100': '#E4F7F3',
+      contrastText: '#FFF',
+    },
+    grayscale: {
+      main: '#8A92A3',
+      max: '#000000',
+      '900': '#242838',
+      '800': '#383E54',
+      '700': '#4A5065',
+      '600': '#6B7384',
+      '500': '#8A92A3',
+      '400': '#A7AFC0',
+      '300': '#C4CAD7',
+      '200': '#DCE0E9',
+      '100': '#EAEDF2',
+      '50': '#F0F3F5',
+      min: '#FFFFFF',
+      contrastText: '#FFF',
+    },
```

Comment:
I think we should name it exaclty as the DS. Designers defined "grayscale" on the same level as "primary" and "secondary", I guess because it's incorrect to say gray to black. Anyways, imo, I think we should change the previous "gray" and use "grayscale" as per definition of the designers on the DS so we have a single language and we we look at the designs, it does not cause confusion.

Reply:
Checked and seems like they are not mapped correctly, so we just use the new approach as is better for the DS.